Description: Fixes invalid transparency in window icon.
 This patch fixes invalid transparency of window icon as
 SDL don't recognize semi-transparency and works only
 with 1bit-transparency. This patch gives a new icon
 with 1bit-transparency (sdl_instead_wnd.png) and
 sets it in SDL_WM_SetIcon() function for correct
 showing of icon in window title and dockbar.
 .
 Note: Upstream author (Peter Kosyh) was already
 informed about this bug. This patch will be
 accepted to upstream in next version.
Author: Sam Protsenko <joe.skb7@gmail.com>
Reviewed-By: <p.kosyh@gmail.com>
Last-Update: <2011-11-11>

--- instead-1.5.2.orig/icon/Makefile.windows
+++ instead-1.5.2/icon/Makefile.windows
@@ -3,3 +3,4 @@ all:
 install:
 	if not exist ..\bin\icon mkdir ..\bin\icon
 	copy sdl_instead.png ..\bin\icon
+	copy sdl_instead_wnd.png ..\bin\icon
--- instead-1.5.2.orig/icon/Makefile
+++ instead-1.5.2/icon/Makefile
@@ -6,6 +6,9 @@ all:
 install:
 	$(INSTALLD) $(ICONPATH)
 	$(INSTALL) sdl_instead.png $(ICONPATH)
+	$(INSTALL) sdl_instead_wnd.png $(ICONPATH)
 
 uninstall:
 	$(RM) $(ICONPATH)/sdl_instead.png
+	$(RM) $(ICONPATH)/sdl_instead_wnd.png
+
--- instead-1.5.2.orig/src/sdl-instead/graphics.c
+++ instead-1.5.2/src/sdl-instead/graphics.c
@@ -1358,7 +1358,7 @@ int gfx_video_init(void)
 #define ICON_PATH "./icon"
 #endif
 
-	icon = IMG_Load( ICON_PATH"/sdl_instead.png" );
+	icon = IMG_Load( ICON_PATH"/sdl_instead_wnd.png" );
 	if ( icon ) {
 		SDL_WM_SetIcon( icon, NULL );
 	}
